### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by art-orient**

### CODE REVIEW

File: src/main/java/school/mjc/stage0/conditions/task1/IntegerComparison.java [hunk 1]
```diff
@@ -4,8 +4,8 @@ public class IntegerComparison {
     public static void main(String[] args) {
         int x = 50;
         int y = 50;
-         (x  y) {
-            System.out.println("Hello World!");
+        _ (x _ y) {
```
Comment (Sukora-Stas) [inline src/main/java/school/mjc/stage0/conditions/task1/IntegerComparison.java]: Could you please describe this changes?

Comment (ChoCoBo1712) [inline src/main/java/school/mjc/stage0/conditions/task1/IntegerComparison.java]: change to double underscores as in the readme or change the readme to include single underscores

**Review by ChoCoBo1712 — CHANGES_REQUESTED** [review]

File: readme.md [hunk 2]
```diff
@@ -37,26 +37,26 @@ code snippet:
 
 
 #### 4) Integer comparison.
-Fill the gaps in the code snippet so as to program compiles and prints: "Hello world!"
+Fill the gaps in the code snippet so as the program compiles and prints: "Hello, World!"
```
Comment (Andrekukshinov) [inline readme.md]: construction here is" so that the "

File: src/main/java/school/mjc/stage0/conditions/task1/IntegerComparison.java (hunk 1 above)
Comment (art-orient) [reply src/main/java/school/mjc/stage0/conditions/task1/IntegerComparison.java]: fixed

File: readme.md (hunk 2 above)
Comment (art-orient) [reply readme.md]: fixed

**Review by ChoCoBo1712 — APPROVED** [review]

**Review by Sukora-Stas — APPROVED** [review]

**Pull request merged**